opengemini_client/client_impl.py

```diff
@@ -0,0 +1,91 @@
+import base64
+from abc import ABC
+from http import HTTPStatus
+
+import requests
+
+from opengemini_client.client import Client
+from opengemini_client.models import Config, Endpoint, RequestDetails
+from opengemini_client.url_const import UrlConst
+from opengemini_client.utils import AtomicInt
+
+
+class ClientImpl(Client, ABC):
+    """
+    OpenGemini client class
+    """
+
+    def __init__(self, config: Config, endpoints: list[Endpoint]):
+        """
+        OpenGemini client
+        :param config:  config object
+        :param endpoints:  urls
+        """
+        self.config = config
+        self.endpoints = endpoints
+        self.prev_idx = AtomicInt(5)
+        self.data_chan = {}
```

Comment:
what's the usage of this variant?

Reply:
accept it. remove prex_idx and data_chan attributes